src/main/kotlin/dev/supergrecko/kllvm/core/typedefs/Builder.kt

```diff
@@ -53,7 +55,38 @@ public class Builder internal constructor(internal val llvmBuilder: LLVMBuilderR
         // TODO: Test
         LLVM.LLVMInsertIntoBuilderWithName(getUnderlyingRef(), instruction.getUnderlyingReference(), name)
     }
+    /**
+     * Create a function call passing in |args| and binding the result into
+     * variable |resultName|. Result discarded if no resultName supplied.
```

Comment:
KDoc uses `[args]` notation to reference things, replace the `|`'s with opening and closing brackets so Dokka can reference things properly

Reply:
Fixed